### DISCUSSION TITLE
How to achieve the effect of ilspycmd programmatically?

### DISCUSSION THREAD

**Discussion by Smartisa** [icsharpcode/ILSpy#2812] [Help]
How should I call ICSharpCode.Decompiler to achieve the effect of this statement?
`./ilspycmd.exe ./target.dll -p -o ./targetProject`

**Comment by siegfriedpammer** [comment]
This is how `ilspycmd` does it:

https://github.com/icsharpcode/ILSpy/blob/42668e810afa2954a3fc3528f6b45a7e191c2a9c/ICSharpCode.ILSpyCmd/IlspyCmdProgram.cs#L277-L288
